CONTRIBUTING.md

```diff
@@ -57,6 +57,7 @@ $ npm publish
 
 # Publish dev archetype in sister directory
 $ cd ../builder-victory-component-dev
+$ git add package.json
```

Comment:
Might want to do `git add .` Or a `git status` then git add...

Readme and other changes get pulled in too...